src/leiningen/monolith.clj

```diff
@@ -122,7 +122,12 @@
   (let [[opts [task & args]] (u/parse-kw-args target/selection-opts args)
         [monolith subprojects] (u/load-monolith! project)
         targets (target/select monolith subprojects opts)
-        profile (plugin/merged-profile (select-keys subprojects targets))]
+        profile (plugin/merged-profile monolith (select-keys subprojects targets))
+        project (reduce-kv
+                  (fn remove-replace-meta
+                    [proj k _v]
+                    (update proj k vary-meta dissoc :replace))
```

Comment:
Why is this needed? 🤔

Reply:
Example: let's say you want to inherit the `:test-paths` vector. In order to have it replace the test paths in a child project, rather than meta-merge them, you need to add the `^:replace` meta to the vector in the parent project. After building the inherited profile, Leiningen will meta-merge the vectors, and it will prefer the left hand side, which is the `:test-paths` vector in the root project.

Thinking about it some more, though—should the output profile even include the paths from the root project? It's not intended to be a "normal" project.

Reply:
I think it currently _does_, but I agree that it doesn't seem useful or common to have sources in those locations. I'd be okay with explicitly removing paths from the metaproject before merging the `:monolith/all` profile.

Reply:
I'd like to keep the scope of this as-is and just introduce the two new inheritance keys and perhaps figure out what to do with this and the missing child dev profiles.